# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/xul/XULTreeGridAccessible.cpp
# Commit: 6d0f4821ddcf
# Full Hash: 6d0f4821ddcfca1b59e4db3befc3ec9c12335891
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-03-23 09:49:32
# Regressor Bug: 1735970
# File Overlap Count: 1
# Description:
#   Bug 1735970 part 3: Add TableAccessibleBase and TableCellAccessibleBase. r=morgan
#   
#   These are needed to support RemoteAccessible tables.
#   Stuff specific to LocalAccessible is still in TableAccessible and TableCellAccessible, which now inherit from the new Base classes.
#   
# ==============================================================================

diff -r d82660fb5408 -r 6d0f4821ddcf accessible/xul/XULTreeGridAccessible.cpp
--- a/accessible/xul/XULTreeGridAccessible.cpp	Wed Mar 23 04:00:25 2022 +0000
+++ b/accessible/xul/XULTreeGridAccessible.cpp	Wed Mar 23 04:00:26 2022 +0000
@@ -62,7 +62,7 @@
   return SelectedItemCount();
 }
 
-void XULTreeGridAccessible::SelectedCells(nsTArray<LocalAccessible*>* aCells) {
+void XULTreeGridAccessible::SelectedCells(nsTArray<Accessible*>* aCells) {
   uint32_t colCount = ColCount(), rowCount = RowCount();
 
   for (uint32_t rowIdx = 0; rowIdx < rowCount; rowIdx++) {
@@ -489,7 +489,7 @@
 uint32_t XULTreeGridCellAccessible::RowIdx() const { return mRow; }
 
 void XULTreeGridCellAccessible::ColHeaderCells(
-    nsTArray<LocalAccessible*>* aHeaderCells) {
+    nsTArray<Accessible*>* aHeaderCells) {
   dom::Element* columnElm = mColumn->Element();
 
   LocalAccessible* headerCell = mDoc->GetAccessible(columnElm);